python_modules/libraries/dagster-k8s/setup.py

```diff
@@ -32,6 +32,6 @@ def get_version() -> str:
         "Operating System :: OS Independent",
     ],
     packages=find_packages(exclude=["dagster_k8s_tests*"]),
-    install_requires=[f"dagster{pin}", "kubernetes"],
+    install_requires=[f"dagster{pin}", "kubernetes<28.1.0"],
```

Comment:
didn't spot anything in https://github.com/kubernetes-client/python/blob/release-28.0/CHANGELOG.md that stood out

don't love adding user effecting pins like this without a good understanding of what broke 

could consider pinning tests specifically like in `integration_tests/test_suites/k8s-test-suite/tox.ini` 

would be good to leave a comment next to the pin explaining why its pinned

Reply:
This did not appear to resolve the issue anyway. https://buildkite.com/dagster/dagster/builds/66435#018aaa08-fb11-43c5-a1e9-8c1239955284